Cargo.toml

```diff
@@ -124,7 +124,7 @@ default = ["cli", "js"]
 
 
 [workspace]
-members = [".", "autogen", "cloudflare", "testconv"]
+members = [".", "autogen", "cloudflare", "lambda", "testconv"]
```

Comment:
```suggestion
members = [".", "autogen", "cloudflare", "aws-lambda", "testconv"]
```

Reply:
Resolved in fedb0e1541ffdf23504c7e0c98ba800b92be3d16